src/index.jsx

```diff
@@ -44,7 +44,8 @@ const AppWrapper = () => (
             <Route exact path="/404" component={NotFoundPage} />
             <Route exact path="/error" component={ErrorPage} />
             <PrivateRoute path="/enterprises" component={EnterpriseIndexPage} />
-            <PrivateRoute path="/:enterpriseSlug" component={App} />
+            <PrivateRoute path="/:enterpriseSlug" component={EnterpriseApp} />
+            <Route exact path="/" component={EnterpriseIndexPage} />
```

Comment:
Other than the rename, this is the only change. We use `exact` here so that other, non-matching URLs still 404 below.